src/infinite-hits/infinite-hits.ts

```diff
@@ -32,7 +32,7 @@ import { noop } from '../utils';
       </div>
 
       <button
-        [class]="cx('showMore')"
+        [class]="cx('loadMore')"
```

Comment:
The class should also support the `disabled` state (see [specs](https://instantsearch-css.netlify.com/widgets/infinite-hits/)).

Reply:
ah ok, thanks for giving me the spec link :)
Out of curiosity, why wasn't `:disabled` enough?

Reply:
Don't ask me why :D -> https://github.com/algolia/instantsearch-specs/issues/19